packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: When in daemon mode, the user would not see imperative initialization errors, but now the errors are passed back to the user terminal window.
```

Comment:
- Is there a pr or issue link for this item?
- Should imperative be capitalized?